.github/workflows/ci-documentbot.yml

```diff
@@ -28,7 +28,7 @@ on:
       - unlabeled
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.pull_request.number }}
```

Comment:
Should use `${{ github.workflow }}-${{ github.ref }}-${{ github.event.pull_request.number }}` as the concurrency group can only use the [github context](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency)

The `github.pull_request.number` can only be retrieved within job.

Reply:
Thank @maxsxu, I updated this value, using the `github.event.number` instead of the `github.pull_request.number`.